tasks/main.yml

```diff
@@ -177,6 +177,13 @@
     group: "{{ tomcat_user_group }}"
     mode: 0640
 
+- name: Remove ROOT Webapp Context
+  file:
+    state: absent
+    path: "{{ tomcat_instance_path }}/webapps/ROOT/"
+  when:
+    - not tomcat_root_context
```

Comment:
I could make  `tomcat_remove_all_default_webapps` that clears all of the default web apps and also include options to keep certain default web apps for example 'tomcat_remove_manager' option.  If I do go this route, I need to ensure those default files are available again, for example in a scenario where `tomcat_remove_all_default_webapps`  is changed to true then false again.